getting-started/try-out-sandbox.md

```diff
@@ -1,17 +1,28 @@
 # Try out CFT Sandbox
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras blandit tristique auctor. Pellentesque pulvinar posuere mi nec dapibus. Nunc imperdiet tortor est, vel molestie turpis accumsan eu. Vestibulum tincidunt ante in consequat consectetur. Praesent at aliquam est. Curabitur vel volutpat urna, et convallis ligula. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+## The Sandbox Environment
+> **Cloud File Transfer (CFT) sandbox environment is a secure environment to try out CFT APIs**.
```

Comment:
Cloud File Transfer (CFT) sandbox is a test environment that is almost identical to our production environment except that it supports only internet to internet transfers. It offers you a space to run file transfers with dummy files, test the API fully, and get a feel of the file transfer process before applying for a production account.